routing.go

```diff
@@ -329,7 +329,7 @@ func (dht *IpfsDHT) getValues(ctx context.Context, key string, stopQuery chan st
 					return nil, err
 				default:
 					return nil, err
-				case nil, errInvalidRecord:
+				case nil, internal.ErrInvalidRecord:
```

Comment:
returning this where the consumer can't match on it seems bad

Reply:
This is returned from an internal function and then swallowed, it shouldn't escape to the user.